studio-docs/source/org/members.mdx

```diff
@@ -7,7 +7,7 @@ A Studio organization can have any number of members, and each member can be ass
 
 ## Organization-wide member roles
 
-> **Billing Mananger and Org Admins are available to all organizations, the other roles are available for organizations with an Enterprise plan.**
+> **Billing Mananger, Consumers and Org Admins are available to all organizations, the other roles are available for organizations with an Enterprise plan.**
```

Comment:
I think you need a second comma in there for the grammarz? `Billing Manager, Consumers, and Org Admins`?